### DISCUSSION TITLE
Controller Navigation

### DISCUSSION THREAD

**Discussion by twisted89** [AvaloniaUI/Avalonia#8535] [Q&A] [answered]
I can't find any references to controller navigation so maybe someone can give me an answer here.

Microsoft documentation refers to "Application.RequiresPointerMode" to enable directional controller navigation support, I couldn't see any way to enable this in Avalonia, Is there an alternative or any reliable way to allow controller navigation in an app? At a minimum, in a Windows and Linux build.

**Comment by jp2masa** [comment, marked as answer]
Not implemented yet: #6945.
